base/src/main/java/io/spine/util/PropertyFiles.java

```diff
@@ -94,17 +89,28 @@ private static Properties loadPropertiesFile(URL resourceUrl) {
             inputStream = resourceUrl.openStream();
             properties.load(inputStream);
         } catch (IOException e) {
-            if (log().isWarnEnabled()) {
-                log().warn("Failed to load properties file from: " + resourceUrl, e);
-            }
+            warn(e, "Failed to load properties file from: %s", resourceUrl);
         } finally {
             close(inputStream);
         }
         return properties;
     }
 
     private static ClassLoader getContextClassLoader() {
-        return Thread.currentThread().getContextClassLoader();
+        return Thread.currentThread()
+                     .getContextClassLoader();
+    }
+
+    private static void warn(Throwable e, String errorFormat, Object... params) {
```

Comment:
This looks a part of some utility.

Reply:
Well spotted. I've moved it to `Logging`.